libs/components/phone-field/src/lib/modules/phone-field/phone-field.component.ts

```diff
@@ -347,20 +348,16 @@ export class SkyPhoneFieldComponent implements OnDestroy, OnInit {
         });
       }
 
-      if (!this.defaultCountry) {
```

Comment:
Moved this value out of the `setTimeout` so that it's immediately available to the phone field input directive.